src/main/java/com/vlkan/pubsub/jackson/JacksonInstantDeserializer.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 Volkan Yazıcı
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permits and
+ * limitations under the License.
+ */
+
+package com.vlkan.pubsub.jackson;
+
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdScalarDeserializer;
+
+import java.io.IOException;
+import java.time.Instant;
+
+public class JacksonInstantDeserializer extends StdScalarDeserializer<Instant> {
```

Comment:
Just FYI, there is a module for Jackson that adds support for `java.time` types

Reply:
Indeed `JavaTimeModule` in `jackson-datatype-jsr310` addresses this issue. But according to my experience in maintaining `log4j2-logstash-layout`, people are superficially allergic to any Jackson dependency. Hence I wanted to keep it at minimum.